brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Admin Dashboard - String updation.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/2634da90-a8d4-4a2c-9d29-a36878a5130f?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- https://share.getcloudapp.com/eDu5gLRL

### Types of changes




- Improvement

### How has this been tested?

- Go to the settings > Perfomance
- Check & verify strings for Display Selected Font Families, Load Google Fonts Locally & Preload Local Fonts.

### Checklist:
